setup.py

```diff
@@ -31,31 +32,22 @@
 try:
     from setuptools import setup, find_packages
     from setuptools import Extension
-    stp = True
-except:
+except ImportError:
     from distutils.core import setup
     from distutils.extension import Extension
-    stp = False
-import _updateversion as up
 
 is_platform_windows = False
 if platform.system() == "Windows":
     is_platform_windows = True
 
-if platform.system() == "Darwin":
-    # make sure you are using Homebrew's compiler
-    os.environ['CC'] = 'gcc-8'
-    os.environ['CXX'] = 'g++-8'
-else:
-    os.environ['CC'] = 'gcc'
-    os.environ['CXX'] = 'g++'
```

Comment:
@Didou09 here we used to force which `gcc` was being used. By default `homebrew`'s gcc is `gcc-8`.
Now we let the system's `gcc`